src/devtools/client/debugger/src/actions/sources/select.ts

```diff
@@ -218,28 +219,13 @@ export function showAlternateSource(
   oldSourceId: string,
   newSourceId: string
 ): UIThunkAction<Promise<void>> {
-  return async (dispatch, getState, { ThreadFront }) => {
+  return async (dispatch, getState) => {
     const state = getState();
     if (getSourceDetails(state, oldSourceId)?.isSourceMapped) {
-      ThreadFront.preferSource(newSourceId, true);
-    } else {
-      ThreadFront.preferSource(oldSourceId, false);
-    }
-
-    let selectSourceByPausing = false;
```

Comment:
:question: what was this `selectSourceByPausing` thing?  Do we still care about that?

Reply:
That's what I meant when I wrote
> the `showAlternateSource()` thunk doesn't need to call the `paused()` thunk anymore: this was necessary in the past to recompute the frames and scopes stored in redux